<?xml version="1.0"?>
<launch>

<!--
  <node pkg="package_name" type="node_name" name="override_name">
    
  </node>

<include file="$(find package_name)/directory/another.launch"/>
-->

<node pkg="geobot_navigation" type="geobot_local_planner_node" name="geobot_local_planner" output="screen">
	<param name="maximum_linear_velocity" value="1.0"/>
	<param name="maximum_angular_velocity" value="1.0"/>
	<param name="error_tolerance2goal" value="0.4"/>
	<param name="no_of_segments_ahead" value="5"/>
	<param name="error_gain" value="1.1"/>
	<param name="cte_gain" value="0.2"/>
	<param name="heading_gain" value="1.0"/>
	<param name="velocity_constraint" value="1.0"/>
        <param name="cross_track_error_deadband" value="0.05"/>	  
	<param name="derivative_gain" value="0.0"/>
	<param name="derivative_heading_gain" value="0.0"/>
	<param name="steering_gain" value="1.0"/>
	<param name="min_delta_time" value="0.1"/> <!-- for D controller -->


  </node>


</launch>

